db/migrate/20150504224641_add_fields_to_projects.rb

```diff
@@ -0,0 +1,5 @@
+class AddFieldsToProjects < ActiveRecord::Migration
```

Comment:
just a general note surrounding this...if you realize before you've already shipped a feature with a database migration that you want to change the table you just created, you're better off doing a db:rollback and modifying the original migration that created the table to just add/remove the columns you want. This way your PR only contains one migration which hold all the necessary info to create your vision of the table at that point in time. Also means less migrations need to be run by Heroku/project-collaborators/you-if-you-ever-need-to-nuke-your-db/etc. Obviously it's a different scenario if you create a table a month ago, do a bunch of other migrations in the meantime, and then need to change something, in which case you do want to just create a new migration rather than rolling back the db a long way and changing the past.